src/components/side_nav/side_nav.a11y.tsx

```diff
@@ -0,0 +1,356 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+/// <reference types="../../../cypress/support"/>
+
+import React, { useState } from 'react';
+import { EuiSideNav } from './side_nav';
+import { EuiIcon } from '../icon';
+import { htmlIdGenerator } from '../../services';
+
+describe('EuiSideNav', () => {
+  describe('Mobile EuiSidenav', () => {
```

Comment:
Very optional nitpick - capitalization on component name, or just skip using the component name altogether
```suggestion
  describe('Mobile EuiSideNav', () => {
```
```suggestion
  describe('Mobile side nav', () => {
```